model/src/main/java/de/evoila/cf/broker/model/context/Context.java

```diff
@@ -0,0 +1,121 @@
+package de.evoila.cf.broker.model.context;
+
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Context {
+
+    @JsonProperty("platform")
+    private String platform;
```

Comment:
Misses an enum or constants to set this field or compare existing values.

Reply:
Just when updating it, or when creating it as well?

Reply:
Also when creating, bad choice of words there from me.
Before setting the context object, there should be a validation check to guarantee the correctness of the context object.

Reply:
This is the case in this implementation.